suites/reef/rados/tier-2_rados_test-pg-split-merge.yaml

```diff
@@ -93,6 +93,63 @@ tests:
         log_to_file: true
       desc: Change config options to enable logging to file
 
+  - test:
+      name: Test Reads Balancer
+      module: test_reads_balancer.py
+      desc: Testing reads balancer functionality in RHCS
+      polarion-id: CEPH-83576078
+      config:
+        online_command_verification: true
+        create_pools:
+          - create_pool:
+              pool_name: rpool_1
+              pg_num: 64
+              byte_size: 256
+              pool_type: replicated
+              rados_write_duration: 50
+              rados_read_duration: 30
+          - create_pool:
+              pool_name: rpool_2
+              pg_num: 128
+              pool_type: replicated
+              rados_write_duration: 50
+              rados_read_duration: 30
+              byte_size: 256
+          - create_pool:
+              pool_name: rpool_3
+              pg_num: 32
+              rados_write_duration: 50
+              rados_read_duration: 30
+              byte_size: 256
+              pool_type: replicated
+          - create_pool:
+              create: true
+              pool_name: ecpool_1
+              pool_type: erasure
+              pg_num: 32
+              k: 4
+              m: 2
+              plugin: jerasure
+              rados_write_duration: 50
+              rados_read_duration: 30
+              byte_size: 256
+          - create_pool:
```

Comment:
Small observation: Since we cannot verify on EC pool the functionalities here but only verifying the proper error message is thrown on EC pool operations may be we can have single EC pool? Any reason we have 2 EC pools created?

Reply:
For the verification of online commands, we just do -ve scenario.

With the verification of offline tool, we want to make sure that applying reads balancing on RE pools does not cause adverse affect on reads of EC pools. hence 2 were created.

But yes, at this point, We can completely do tests with single EC pool as well.

Will do as suggested!

Reply:
Removed 1 EC pool creation.